### PR TITLE
Updates template based on cinder-solidfire and cinder-nimblestorage

### PR DESCRIPTION
**Pull request by gustavosr98**

### CODE REVIEW

**Review by lourot — APPROVED** [review]

File: cinder-{{cookiecutter.driver_name_lc}}/osci.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+- project:
+    templates:
+      - charm-unit-jobs-py38
+      - charm-unit-jobs-py39
```
Comment (lourot) [inline cinder-{{cookiecutter.driver_name_lc}}/osci.yaml:4]: This doesn't run Zaza at all, see https://review.opendev.org/c/openstack/charm-cinder-nimblestorage/+/825162

**Review by lourot — CHANGES_REQUESTED** [review]

File: cinder-{{cookiecutter.driver_name_lc}}/tests/bundles/{{cookiecutter.ubuntu_release}}-{{cookiecutter.openstack_release}}.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+series: bionic
```
Comment (ChrisMacNaughton) [inline cinder-{{cookiecutter.driver_name_lc}}/tests/bundles/{{cookiecutter.ubuntu_release}}-{{cookiecutter.openstack_release}}.yaml]: this should probably be `series: {{cookiecutter.ubuntu_release}}`

File: cinder-{{cookiecutter.driver_name_lc}}/osci.yaml (hunk 1 above)
Comment (lourot) [reply cinder-{{cookiecutter.driver_name_lc}}/osci.yaml:4]: discussed with @ChrisMacNaughton: this is probably OK as usually when creating a new cinder charm, we won't have a backend appliance ready in our CI system at first

**Review by lourot — APPROVED** [review]

**Review by ChrisMacNaughton — COMMENTED** [review]
generally +1 with one nit

File: cinder-{{cookiecutter.driver_name_lc}}/osci.yaml (hunk 1 above)
Comment (ChrisMacNaughton) [reply cinder-{{cookiecutter.driver_name_lc}}/osci.yaml:4]: ideally, part of accepting a new charm that has been written with this cookiecutter is documented manual validation that _has_ been run against the hardware in question, though :)

**Review by lourot — APPROVED** [review]

**Pull request merged**